### ISSUE TITLE
[BE] 10-10EZR Form: SEND all EZR data to ES API

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#63724]
As a Veteran, I would like to be able to update as much of my information online as I can so that I can make changes easily without having to call the hotline and wait on hold or print out the form and snail mail it in.

- [x] Build an API that can SEND all EZR fields to the ES (EZR Sections I-VII) (fields listed below)
- [ ] Pass back information that is not displayed*
- [x] Send information synchronously to VES
- [ ] Send email when it fails (Josh F. will control confirmation email)
- [ ] Send alert when it fails
- [ ] Determine how to handle required fields (do we have to send all required info? If so, how would we do that?)
- [x] Test API to make sure all info it sent to VES
- [ ] **Success: We have an API that can send all EZR info from our app to ES**

--------------------------------------------------
EZR Fields ([spreadsheet with all EZR fields](https://docs.google.com/spreadsheets/d/1IGLA8GT35ffS2PnFvCwdnA7bWTL5187TN9QlVwUXYr4/edit#gid=1410739912)):

**Section I - General Information**
Name (Read only for Veteran)
Preferred Name (will be behind feature flag on front end--VES cannot receive) 
Birth Sex (Read only for Veteran)
Gender Identity (will be behind feature flag on front end)
Social Security Number (Read only for Veteran)
Date of Birth (Read only for Veteran)
Mailing Address 
Home Telephone Number 
Mobile Telephone Number
E-mail Address 
Home Address 
Current Martial Status 

**Section II - Insurance Information**
Health Insurance Company Name
Name of Policyholder 
Policy Number 
Group Code 
Eligible for Medicaid? 
Enrolled in Medicare Hospital Insurance Part A? 
Medicare Part A effective date?
Medicare number

**Section III - Dependent Information**
Spouse's Name
Spouse's SSN
Spouse's DOB 
Date of Marriage
Do you and your spouse cohabitatate? Y/N
(if no) Spouse's Address
Dependent's Name
Dependent's DOB
Dependent's SSN
Date dependent became dependent 
Dependent's relationship to you 
Is your dependent living with a permanent disability that happened before they turned 18 years old? 
If your dependent is between 18 and 23, were they enrolled as a student in 20xx? 
Total amount of money your dependent paid for college, vocational rehabilitation or training? 
Did your dependent live with you in 20xx? (previous year)
Did your dependent earn income in 20xx? (previous year)

**Section IV - Previous Year Gross Annual Income**
Gross Income from work (Veteran, Spouse, Dependent) 
Net income from a farm, property, or business (Veteran, Spouse, Dependent)
Other Incomes (Veteran, Spouse, Depedent)

**Section V - Previous Year Deductible Expenses**
Non-reimbursable medical expenses
Funeral and burial expenses for a spouse or depended who died 
College or vocational education expenses

-----------------------------------------------
**Resources:**

- **[API Docs](https://ves.va.gov/esr-ws/spring-ws/getEESummary/eeSummary.wsdl)**
- [Lihan's Documentation](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/tutorials/record_vcr.md)
- [VA Profile API sharepoint](https://dvagov.sharepoint.com/sites/OITEPMOVAPROPUB/SitePages/Getting-Started.aspx)
- [Health Benefits Swagger Doc](https://qa.vaprofile.va.gov:7005/health-benefit/swagger-ui/index.html?configUrl=/health-benefit/v3/api-docs/swagger-config)
- [Personal Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/personal_information/backend_documentation.md)
- [Contact Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/contact_information/backend_documentation.md)
- [Military Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/military_info/backend_documentation.md)
- [Direct Deposit Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/direct_deposit/backend_documentation.md)
- [Notification Settings](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/notification_settings/backend_documentation.md)
- [Account Security](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/account_security/backend_documentation.md)
